judoscale-ruby/lib/judoscale/config.rb

```diff
@@ -89,9 +89,11 @@ def reset
         @api_base_url ||= "https://adapter.judoscale.com/api/#{ENV["RENDER_SERVICE_ID"]}"
       elsif ENV["DYNO"]
         @current_runtime_container = RuntimeContainer.new ENV["DYNO"]
+      elsif (metadata_uri = ENV["ECS_CONTAINER_METADATA_URI"])
+        @current_runtime_container = RuntimeContainer.new(metadata_uri.split("/").last)
```

Comment:
Ah, I totally missed this PR email and read my emails in backwards order. This makes more sense now!